src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -328,93 +328,193 @@ public static void SupportNonNullableReferenceTypes(this SwaggerGenOptions swagg
         /// <summary>
         /// Extend the Swagger Generator with "filters" that can modify Schemas after they're initially generated
         /// </summary>
-        /// <typeparam name="TFilter">A type that derives from ISchemaFilter</typeparam>
+        /// <typeparam name="TFilter">A type that derives from <see cref="ISchemaFilter"/></typeparam>
         /// <param name="swaggerGenOptions"></param>
         /// <param name="arguments">Optionally inject parameters through filter constructors</param>
         public static void SchemaFilter<TFilter>(
             this SwaggerGenOptions swaggerGenOptions,
             params object[] arguments)
             where TFilter : ISchemaFilter
         {
+            swaggerGenOptions = swaggerGenOptions ?? throw new ArgumentNullException(nameof(swaggerGenOptions));
             swaggerGenOptions.SchemaFilterDescriptors.Add(new FilterDescriptor
             {
                 Type = typeof(TFilter),
                 Arguments = arguments
             });
         }
 
+        /// <summary>
+        /// Extend the Swagger Generator with "filters" that can modify Schemas after they're initially generated
+        /// </summary>
+        /// <typeparam name="TFilter">A type that derives from <see cref="ISchemaFilter"/></typeparam>
+        /// <param name="swaggerGenOptions"></param>
+        /// <param name="filterInstance">An instance of the filter, to enable reuse.</param>
```

Comment:
I think this a simpler way of phrasing it.

```suggestion
        /// <param name="filterInstance">The filter instance to use.</param>
```